### ISSUE TITLE
rgbpm2 doesn't work anymore seemly since Ubuntu 20.04.1 SOLVED

### ISSUE THREAD

**Issue by Eduardo-EazyEZ** [Moonbase59/loudgain#40]
Thank you for this awesome program, I believe yesterday my machine was updated from Ubuntu 20.04 to 20.04.1 I am not really sure. On Monday my /home disk got somehow locked without me noticing, not knowing any better I reinstalled and since then no matter where I put https://github.com/Moonbase59/loudgain/blob/master/bin/rgbpm2 be it /home/e/bin or /usr/local/bin or run it from the git clone /home/e/Software/loudgain/bin it doesn't work anymore where just a few days ago it did. rgbpm on the other hand works just fine, this is the output i get from running rgbpm2
![Screenshot from 2020-12-22 20-22-25](https://user-images.githubusercontent.com/59966270/102949732-f4eeb580-4496-11eb-963a-9c3b69563d00.png)

**Comment by Moonbase59** [comment]
Hi there! Looks like it cannot find `python`. Did you install it? Or is Python 2.x now finally being removed?

**Comment by Eduardo-EazyEZ** [comment]
Hi!
Thank you for the speedy answer. As I was having the same trouble with other python apps it totally fell of my mind I had opened this issue with loudgain.
You are right python2 is no more and wether you install it (I did) or not you'd get the same output. I found my solution at https://askubuntu.com/questions/942930/usr-bin-env-python-no-such-file-or-directory and picked the second offered solution which worked like a charm. For Ubuntu 20.04 just:

`sudo apt-get install python-is-python3`

Sorry I forgot to flga this solved.
Merry Christmas if that is your thing and a happy New Year again if you count like I do ;)
